hello-world-dotnet-task/readme.md

```diff
@@ -0,0 +1,27 @@
+# Hello World .NET
+
+Sample for Hello World task. Print "Hello World" when run the main.py
+
+### Prerequisites
+1. Fork the repositoy
+
+## Getting started
+
+Please refer to the Choreo documentation under the [Develop an Application with Buildpacks](https://wso2.com/choreo/develop-components/deploy-an-application-with-buildpacks) to learn how to deploy the application.
+
+1. Select `Manual Task` Card from Component Creation Wizard
+2. Select `.NET` as the buildpack. Fill as follow according to selected Buildpack.
+
+    | **Field**             | **Description**                               |
+    |-----------------------|-----------------------------------------------|
+    |Name           | Hello World Dotnet Task              |
+    |Description    | Hello World Dotnet Task       |
+    | **GitHub Account**    | Your account                                  |
+    | **GitHub Repository** | choreo-samples |
+    | **Branch**            | **`main`**                               |
+    | **Buildpack**      | .NET |
+    | **Select Go Project Directory**       | hello-world-python-task |
```

Comment:
lets update this as well

Reply:
Fixed with https://github.com/wso2/choreo-samples/pull/101/commits/8c0bf8690a2860b6f806d4bcad197fe86f1792c7